flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/calcite/PreValidateReWriter.scala

```diff
@@ -330,7 +331,7 @@ object PreValidateReWriter {
         if (currentNodes.size > 0) {
           fieldNodes.addAll(currentNodes)
         }
-        fixedNodes.add(SqlStdOperatorTable.ROW.createCall(value.getParserPosition, fieldNodes))
+        fixedNodes.add(FlinkSqlOperatorTable.ROW.createCall(value.getParserPosition, fieldNodes))
```

Comment:
Are you sure this change is enough to replace `SqlStdOperatorTable.ROW`? IIRC the generated code in the parser usually has `SqlStdOperatorTable` hard coded. Can you verify? If this is the case we need to shadow the Calcite class instead.